README.md

```diff
@@ -419,6 +423,8 @@ exports.handler = (event, context, callback) => {
 Each application should be unit-tested. We are exposing simple API helping you to create sample Alexa requests for testing and debugging
 
```

Comment:
Add the testing dependencies installation here:
"
Example below illustrates simple unit testing example for `intentRequest`. Testing of `launchRequest` or `sessionEndedRequest` would look the same

```
npm install mocha chai expect --save-dev
```
"